SDWebImage/Core/SDWebImageManager.m

```diff
@@ -341,6 +342,10 @@ - (void)callStoreCacheProcessForOperation:(nonnull SDWebImageCombinedOperation *
                     } else {
                         cacheData = (imageWasTransformed ? nil : downloadedData);
                     }
+                    // keep the original image format and extended data
```

Comment:
This is not correct. The transformed image should have the totally different from the original one, one any attributes should be inherited.